js/catchTheFlow.js

```diff
@@ -627,9 +629,9 @@ function displayLeaves()
 
 function pauseAndPlay()
 {
-	if(pauseState  === 0)
```

Comment:
There is a problem with this change.(I checked it) You're able to pause the game using the pause button on the screen, but unable to resume it. And replacing pauseState with game.Paused is unnecessary here(in the way you changed it).